interface/main/tabs/main.php

```diff
@@ -204,6 +216,27 @@ function isEncounterLocked( encounterId ) {
            $('#attendantData').slideToggle();
             $('#patient_caret').toggleClass('fa-caret-down').toggleClass('fa-caret-up');
         });
+        
+        //Javascript for the tab drag and drop
+        $('.sorting').sortable({
```

Comment:
Hello, @MatthewVita  
yes, makeTabsSortable its a same code for making tab  sortable. But we have made new function in the interface\main\tabs\js\tabs_view_model.js function 

Thanks
Rishabh Software